hawk/app/views/shared/_simulator.html.haml

```diff
@@ -15,23 +15,22 @@
             .col-sm-12
               .errors
           .row
-            .col-sm-7
-              %select#sim-events.form-control{multiple: true}
-              .row
-                &nbsp;
-              %button#sim-addnode.btn.btn-default.btn-sm{type: :button}
-                = icon_text "plus", _("Node")
-              %button#sim-addop.btn.btn-default.btn-sm{type: :button}
-                = icon_text "plus", _("Operation")
-              - unless current_cib.tickets.empty?
-                %button#sim-addticket.btn.btn-default.btn-sm{type: :button}
-                  = icon_text "plus", _("Ticket")
-              %button#sim-delete.btn.btn-default.btn-sm{type: :button}
-                = icon_tag "minus"
-            .col-sm-5
-              .well
-                %button#sim-results.btn.btn-default{type: :button, disabled: :disabled}
-                  = _("Results")
+            %select#sim-events.form-control{multiple: true}
+            .row
+              &nbsp;
+            %button#sim-addnode.btn.btn-default.btn-sm{type: :button}
```

Comment:
Classes should be listed before IDs (.btn should precede #sim-addnode)<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace